_data/cfpb-brand-colors.csv

```diff
@@ -5,48 +5,56 @@ green,green-80,Green 80,#66c368,"102, 195, 104",,"55, 0, 76, 0",,
 green,green-60,Green 60,#addc91,"173, 220, 145",358,"32, 0, 52, 0",TRUE,
 green,green-40,Green 40,#c7e5b3,"199, 229, 179",,"22, 0, 36, 0",,
 green,green-20,Green 20,#e2efd8,"226, 239, 216",361 @ 20%,"12, 0, 19, 0",TRUE,
+green,green-10,Green 10,#f0f8eb,"240, 248, 235",,,,

```

Comment:
Are the number of trailing commas here correct? The line above has a set of commas in quotes.

Reply:
Yes, they're correct. These shades won't have corresponding PMS colors, we haven't yet determined CMYK equivalents for these shades, and they are not considered primary or secondary colors.